app/modules/users/views/templates/contribution.hbs

```diff
@@ -25,6 +29,12 @@
       </div>
     {{/with}}
   </div>
+  {{#if showUser}}
+    <a class="user showUserContributions" href="{{user.contributions}}">
+      <img src="{{imgSrc user.picture 32}}" alt="{{user.username}}">
+      <p class="username">{{user.username}}</p>
+    </a>
+  {{/if}}
   <p class="time" title="{{localTimeString timestamp}}">{{timeFromNow timestamp}}</p>
```

Comment:
Since this is mainly for log changes, i would be in favor of a more precise date format

manual suggestion (did the gh feature disappear?):

```hbs
<p class="time" title="{{timeFromNow timestamp}}">{{localTimeString timestamp}}</p>
```

Reply:
suggestion still here:
```suggestion
  <p class="time" title="{{timeFromNow timestamp}}">{{localTimeString timestamp}}</p>
```

Reply:
done in 0d39c0a5